etcdserver/cluster_util.go

```diff
@@ -174,6 +228,12 @@ func getVersion(m *Member, tr *http.Transport) (string, error) {
 		if err != nil {
 			continue
```

Comment:
for each member, it might have several endpoints. so we try them one by one. if all of them fails, we return the last error encountered at line248.

what is your suggestion? do you want to log all of them?